db_stress_tool/expected_value.h

```diff
@@ -37,13 +37,13 @@ class ExpectedValue {
   explicit ExpectedValue(uint32_t expected_value)
       : expected_value_(expected_value) {}
 
-  bool Exists() const { return PendingWrite() || !IsDeleted(); }
+  bool Exists() const { return !IsDeleted(); }
```

Comment:
Can we assert(!PendingWrite()) here since we assume there's no concurrent writes?

Reply:
fixed